nncf/quantization/layers.py

```diff
@@ -448,11 +471,7 @@ def _prepare_export_quantization(self, x: torch.Tensor):
             level_low = self.level_low
             level_high = self.level_high
             if self._half_range:
-                if self.scale_shape[0] > 1:
-                    for i in range(self.scale_shape[0]):
-                        x[i] = torch.clamp(x[i], min=input_low[i].item(), max=input_high[i].item())
-                else:
-                    x.data = torch.clamp(x, min=input_low.item(), max=input_high.item())
+                x = torch.min(torch.max(x, input_low), input_high)
```

Comment:
@aleksei-kashapov FYI, the tests are passing with this change in place. However, I remember you having some trouble with this before. Do you recall exactly why the min-max version of clamp could not be used originally?

Reply:
Cause torch.clamp works only in per-tensor mode

Reply:
This is incorrect formula. You do clamp only in per-tensor mode, while it can be per-channel

Reply:
Okay, I rechecked it again and figured out that torch.min and torch.max also work in per-channel mode. 
In that case this formula is correct and much better than I had before